### PR TITLE
Removed unnecessary  "

### PR DESCRIPTION
**Pull request by Patel-Muhammad**
I have removed the " which was added by mistake.

### CODE REVIEW

File: Google-Summer-of-Code-2023.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ We've also asked our previous GSoC contributors what they learned during previou
       - It is really an awesome experience working with some amazing folks from all around the world at Oppia.
       - The kind of support the complete community provides is extra-ordinary.
     - Giving back
-      - "Everybody wants to do some good work but lets be honest, it is maybe the lazyness or the time issue that you are not able to do the type of work you always wanted to do, what can be a good way to give back to the community than to contributing here.
+      - Everybody wants to do some good work but lets be honest, it is maybe the lazyness or the time issue that you are not able to do the type of work you always wanted to do, what can be a good way to give back to the community than to contributing here.
```
Comment (seanlip) [inline Google-Summer-of-Code-2023.md]: lets --> let's

**Review by seanlip — COMMENTED** [review]
Thanks, @Patel-Muhammad! Just one more small thing I noticed, would you be able to fix it please?

File: Google-Summer-of-Code-2023.md (hunk 1 above)
Comment (Patel-Muhammad) [reply Google-Summer-of-Code-2023.md]: @seanlip sure I have fixed it.

**Review by seanlip — APPROVED** [review]
Thanks @Patel-Muhammad! Merging.

**Pull request merged**